src/Validator/EventListener/ValidateListener.php

```diff
@@ -53,7 +53,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event): void
 
         if (
             $controllerResult instanceof Response
-            || $request->isMethodSafe(false)
+            || $request->isMethodSafe()
```

Comment:
I had put isCacheable here, wasn't it correct?

Reply:
No. We need to exclude all safe methods, not just cacheable ones.